services/core-web/common/selectors/staticContentSelectors.js

```diff
@@ -274,8 +274,22 @@ export const getDropdownIncidentStatusCodeOptions = createSelectorWrapper(
 
 export const getDropdownIncidentCategoryCodeOptions = createSelectorWrapper(
   getIncidentCategoryCodeOptions,
-  createDropDownList,
-  ["description", "mine_incident_category_code", "active_ind"]
+  (options) => {
+    return options.map((item) => {
+      return {
+        value: item.mine_incident_category_code,
+        label: item.description,
+        isActive: !item.is_historic,
+        subType: item.parent_mine_incident_category_code ?? null,
+      };
+    });
+  },
+  [
+    "description",
+    "mine_incident_category_code",
+    "is_historic",
+    "parent_mine_incident_category_code",
+  ]
```

Comment:
Will "description" map to "item.mine_incident_category_code" ?

Reply:
Oops, that wasn't quite right.
is_historic and is_active are basically opposite, so the custom mapping function was still necessary. However, custom mapping function doesn't require a parameter list like createDropdownList does, so the array can be removed (it wasn't doing anything).